templates/vue-demo-store/pages/account/order.vue

```diff
@@ -85,6 +85,7 @@ await useAsyncData("getOrders", () => {
   return loadOrders({
     limit: limit.value,
     page: route.query.p ? Number(route.query.p) : defaultPage,
+    checkPromotions: true, // determines whether "paymentChangeable" list is available in the response
```

Comment:
We should use here `orderAssociations`, because currently we have 2 ways of loading it

https://github.com/shopware/frontends/blob/main/packages/composables/src/useOrderDetails/useOrderDetails.ts#L140
https://github.com/shopware/frontends/blob/main/packages/composables/src/useDefaultOrderAssociations/useDefaultOrderAssociations.ts#L42

Reply:
"yes and no" 😄 
 maybe we could create a different helper for params in general because the `checkPromotion` flag is not an association, but the root level param.

Reply:
But now, you cannot change this value by the injection. This property can affect the response time. So it is good to have a flexibility in it

Reply:
the layout in the template is fully customizable and the useOrderDetails is for logged-in users only, not that bad imo so far. 
I'm not sayin that keeping `checkPromotion` or other params in adjustable composable is bad, but for now the composable is designed for associations. I will create a ticket for that case, wdyt?

Reply:
https://github.com/shopware/frontends/issues/1493